src/ucp/core/ucp_request.c

```diff
@@ -44,8 +44,17 @@ void ucp_request_release(void *request)
     ucs_assert(!(req->flags & UCP_REQUEST_FLAG_EXTERNAL));
 
     if ((req->flags |= UCP_REQUEST_FLAG_RELEASED) & UCP_REQUEST_FLAG_COMPLETED) {
+        ucp_worker_h UCS_V_UNUSED worker;
+        ucs_mpool_t *mp = ucs_mpool_obj_owner(req);
```

Comment:
not critical - but can avoid the variable mp and do it shorter:
` ucp_worker_h UCS_V_UNUSED worker = ucs_container_of(ucs_mpool_obj_owner(req, ucp_worker_t, req_mp);`